src/main/scala/org/allenai/plugins/ReleasePlugin.scala

```diff
@@ -62,11 +63,43 @@ object ReleasePlugin extends AutoPlugin {
     }
 
     def settings: Seq[Def.Setting[_]] = Seq(
-      releaseVersion := { version => SemanticVersion.computeReleaseVersion(version) },
-      nextVersion := { version => SemanticVersion.computeNextVersion(version) }
+      ReleaseKeys.releaseVersion := { version => SemanticVersion.computeReleaseVersion(version) },
+      ReleaseKeys.nextVersion := { version => SemanticVersion.computeNextVersion(version) }
     )
   }
 
-  override lazy val projectSettings: Seq[Def.Setting[_]] =
-    WrappedReleasePlugin.releaseSettings ++ DateVersion.settings
+  val checkBranchIsNotMaster = { st: State =>
+    val vcs = Project.extract(st).get(ReleaseKeys.versionControlSystem).getOrElse {
+      sys.error("Aborting release. Working directory is not a repository of a recognized VCS.")
+    }
+
+    if (vcs.currentBranch == "master") {
+      sys.error("Current branch is master.  At AI2, releases are done from another branch and " +
```

Comment:
This specifies which branch CANNOT be used for releases.  Setting is a good idea--I'll see how easy that is for me.